src/commands/draft/draftCommands.ts

```diff
@@ -235,8 +235,8 @@ async function getRepo(octokit: Octokit, remote: Remote): Promise<GitHubRepo | n
 
     let response: RestEndpointMethodTypes["repos"]["get"]["response"];
     try {
-        response = await octokit.rest.repos.get({ owner, repo });
-    } catch (e) {
+        response = await octokit.repos.get({ owner, repo });
```

Comment:
Think the change back to "octokit.repos.get" might be a mistake.

Reply:
lemme quickly fix this. nice!